docs/manage/configuration.md

```diff
@@ -350,7 +327,8 @@ If you run into issues, here are some things to try:
 
 ## Local Setup
 
-Configuring `viam-server` with the Viam app allows you to make use of the cloud features of Viam:
+If you need to configure a robot that will never connect to the internet, you can manually create a local config file on your robot.
+Rather than following the steps on the Viam app **setup** tab to put config fetching credentials in that file, you'll need to paste the full raw JSON config for your robot into your robot's <file>/etc/viam.json</file> file.

```

Comment:
I think you can add this sentence below after the existing `If you are configuring a robot that can never connect to the internet, you can manually create a local full raw JSON config file on your robot.`

Reply:
Done, but I'm unsure if I added this correctly-- lmk if there is an issue here @npentrel